tests/ClientBuilderTest.php

```diff
@@ -161,12 +161,13 @@ public function testGetClient(): void
      */
     public function testIntegrationsAreAddedToClientCorrectly(bool $defaultIntegrations, array $integrations, array $expectedIntegrations): void
     {
-        $clientBuilder = new ClientBuilder([
-            'default_integrations' => $defaultIntegrations,
-            'integrations' => $integrations,
-        ]);
+        $options = new Options();
+        $options->setDefaultIntegrations($defaultIntegrations);
+        $options->setIntegrations($integrations);
 
+        $clientBuilder = new ClientBuilder($options);
```

Comment:
We can just use `ClientBuilder::create` and get the client in one-line instead of saving the instance of the builder which we don't use anymore in a temporary variable